app/services/booking_responder/reject.rb

```diff
@@ -1,44 +1,9 @@
 class BookingResponder
```

Comment:
When the visitors are banned / not on list the rejections reasons are: not enough adults, visitor banned and visitor not on list. We need to rethink what the rejection reasons should be, one to discuss with @kyliemulholland